gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -31,6 +31,22 @@ class LaunchConfigTask extends DefaultTask {
 
     static final List<String> tmpdirJvmOpts = ['-Djava.io.tmpdir=var/data/tmp']
 
+    static final Map<String, List<String>> gcProfiles = [
+            'throughput'   : [],
```

Comment:
This means we would use jvm default which switched to G1 in 9. Need to put correct settings but at the same time java releases claim that from 10 onwards G1 should supersede parallelold collector